tutorials/tutorial-hello-timescale.md

```diff
@@ -0,0 +1,798 @@
+(5, 'negotiated fare'),
+(6, 'group ride');
+```
+
+You can confirm the success of these scripts by running the `\dt` command
+in the `psql` command line. You should see the following:
+
+```sql
+           List of relations
+ Schema |     Name      | Type  |  Owner
+--------+---------------+-------+----------
+ public | payment_types | table | postgres
+ public | rates         | table | postgres
+ public | rides         | table | postgres
+(3 rows)
+```
+
+#### Load trip data into Timescale Cloud
+
+Next, let’s upload the taxi cab data into your Timescale Cloud instance.
+The data is in the file called `nyc_data_rides.csv` and we will load it 
+into the `rides` hypertable. To do this, we’ll use the `psql` `\copy` command below. 
+
+>:WARNING: This dataset is quite large and this command may take a while, depending on the speed of your Internet connection.
+
+```sql
+\COPY rides FROM nyc_data_rides.csv CSV;
+```
+
+You can validate your setup by running the following command:
+
+```sql
+SELECT * FROM rides LIMIT 5;
+```
+
+If you see something similar to the following result, congrats
+you’ve successfully completed Mission 1!
+
+```sql
+-[ RECORD 1 ]---------+--------------------
+vendor_id             | 1
+pickup_datetime       | 2016-01-01 00:00:01
+dropoff_datetime      | 2016-01-01 00:11:55
+passenger_count       | 1
+trip_distance         | 1.20
+pickup_longitude      | -73.979423522949219
+pickup_latitude       | 40.744613647460938
+rate_code             | 1
+dropoff_longitude     | -73.992034912109375
+dropoff_latitude      | 40.753944396972656
+payment_type          | 2
+fare_amount           | 9
+extra                 | 0.5
+mta_tax               | 0.5
+tip_amount            | 0
+tolls_amount          | 0
+improvement_surcharge | 0.3
+total_amount          | 10.3
+-[ RECORD 2 ]---------+--------------------
+vendor_id             | 1
+pickup_datetime       | 2016-01-01 00:00:02
+dropoff_datetime      | 2016-01-01 00:11:14
+passenger_count       | 1
+trip_distance         | 6.00
+pickup_longitude      | -73.947151184082031
+pickup_latitude       | 40.791046142578125
+rate_code             | 1
+dropoff_longitude     | -73.920768737792969
+dropoff_latitude      | 40.865577697753906
+payment_type          | 2
+fare_amount           | 18
+extra                 | 0.5
+mta_tax               | 0.5
+tip_amount            | 0
+tolls_amount          | 0
+improvement_surcharge | 0.3
+total_amount          | 19.3
+```
+
+### Mission 2: Analysis
+
+Let's say that the NYC Taxi and Limousine Commission has made it a key
+goal to mitigate the impact of global warming by reducing their greenhouse 
+gas emissions by 20% by 2024. Given the number of taxi rides taken each 
+day, they believe studying past taxi rider history and behavior will enable 
+them to plan for the future.
+
+To start, you will make sense of historical taxi ride data, draw insights, and 
+use your analysis to plan for the future. In this tutorial, we will limit analysis 
```

Comment:
'plan for the future' is repeated in these two paragraphs in close proximity.  Consider rewording